.circleci/config.yml

```diff
@@ -0,0 +1,38 @@
+version: 2
+
+jobs:
+  build:
+    docker:
+      - image: cimg/python:3.7
+    steps:
+      - checkout
+
+      - restore_cache:
+          keys:
+            - deps-{{ checksum "poetry.lock" }}
+
+      - run:
+          name: Install Dependencies
+          command: poetry install
+
+      - save_cache:
+          key: deps-{{ checksum "poetry.lock" }}
+          paths:
+            - /home/circleci/.cache/pypoetry/virtualenvs
+
+      - run:
+          name: Run pylint
+          command: poetry run pylint symphony_api_client_python tests
```

Comment:
Regarding the idea renaming the package to `symphony.bdk`, should we consider to rename it within this PR so we won't need to update the circleCI jobs in future?